test/provisioner/provision_ibmcloud.go

```diff
@@ -254,6 +254,40 @@ func deleteVpcImpl() error {
 	return deleteVPC()
 }
 
+func isClusterReady(clrName string) (bool, error) {
+	target := containerv2.ClusterTargetHeader{
+		Provider: "vpc-gen2",
+	}
+	clusters, err := IBMCloudProps.ClusterAPI.List(target)
+	if err != nil {
+		return false, err
+	}
+	for _, cluster := range clusters {
+		//ibmcloudTrace(fmt.Sprintf("checking cluster %s in list.\n", cluster.Name))
+		if cluster.Name == clrName && strings.EqualFold(cluster.State, "normal") {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func foundCluster(clrName string) (bool, error) {
+	target := containerv2.ClusterTargetHeader{
+		Provider: "vpc-gen2",
+	}
+	clusters, err := IBMCloudProps.ClusterAPI.List(target)
+	if err != nil {
+		return false, err
+	}
+	for _, cluster := range clusters {
+		//ibmcloudTrace(fmt.Sprintf("checking cluster %s in list.\n", cluster.Name))
```

Comment:
if this line not need we should remove it?

Reply:
done, thanks!